src/types/dc_msg.ts

```diff
@@ -17,6 +18,21 @@ export type DCMessageJitter = number;
 export type TrackInfo = {
     type: string;
     sender_id: string;
+    mime_type: CodecMimeType;
 }
 export type DCMessageMediaMap = {[key: string]: TrackInfo};
 
+export enum CodecSupportLevel {
+    None = 0,
+    Partial = 1,
+    Full = 2,
+}
```

Comment:
Technically we could have made this a boolean since we only care about `None` and `Full` but I thought about it and felt this was easier to extend or leverage if we change our logic, for whatever reason.